Automattic/newspack-plugin
# Remove Unused Wizards

### All Submissions:






### Changes proposed in this Pull Request:

Removes the individual Revenue Wizards which have been combined into the unified Reader Revenue Wizard: 

- Donations
- Memberships Page
- Reader Revenue Onboarding
- Subscriptions

### How to test the changes in this Pull Request:

1. Checkout the branch, run `npm run build:webpack`
2. Verify no errors in the build process.
3. Verify Revenue features work exactly as before.

### Other information:
